### ISSUE TITLE
[!] (plugin svelte) TypeError: Cannot read property 'trim' of undefined

### ISSUE THREAD

**Issue by DhyeyMoliya** [sveltejs/svelte-preprocess#235]
**Describe the bug**
When I auto format Svelte file in VS Code, it changes the spacings and structure of code a little bit. And when I build it gives this error 
`[!] (plugin svelte) TypeError: Cannot read property 'trim' of undefined`

**Logs**
![build-error](https://user-images.githubusercontent.com/6321840/91813518-76a0a600-ec50-11ea-9e43-db9646ffec90.png)

![error](https://user-images.githubusercontent.com/6321840/91813505-6e486b00-ec50-11ea-8c99-d8fff56ea346.png)

**To Reproduce**
The `.svelte` file before formatting (works perfectly) :
![not-formatted](https://user-images.githubusercontent.com/6321840/91813640-a2239080-ec50-11ea-873e-8cb9093cd0e2.png)

The `.svelte` file after formatting (gives build error) :
![formatted](https://user-images.githubusercontent.com/6321840/91813727-bff0f580-ec50-11ea-86bb-ecbbe007bd57.png)

**Expected behavior**
It should work in both scenarios. Because the auto-formatting should not be a problem.

**Stacktraces**
<details>
<summary>Stack trace</summary>

```
bundles src/main.ts → public/build/bundle.js...
[!] (plugin svelte) TypeError: Cannot read property 'trim' of undefined
src/main/main.svelte
TypeError: Cannot read property 'trim' of undefined
    at Object.exports.getTagInfo (/Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/svelte-preprocess/dist/modules/tagInfo.js:25:35)
    at /Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/svelte-preprocess/dist/autoProcess.js:80:93
    at Object.transformMarkup (/Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/svelte-preprocess/dist/modules/markup.js:25:45)
    at markup (/Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/svelte-preprocess/dist/autoProcess.js:114:25)
    at preprocess (/Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/svelte/src/compiler/preprocess/index.ts:86:27)
    at Object.transform (/Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/rollup-plugin-svelte/index.js:221:26)
    at /Users/dhyeymoliya/Documents/developer/svelte-app/node_modules/rollup/dist/shared/rollup.js:18615:25
```
</details>

**Information about your project:**

- Your browser and the version: `N/A`

- Your operating system: `OS X 10.15.6`

- `svelte-preprocess` version : `4.0.0`

- Project uses or Rollup

**Additional context**
I am using Preprocessor and distributing my `.css`, `.js` and `.html` files outside the `.svelte` file.

**Comment by DhyeyMoliya** [comment]
A temporary workaround for me at this moment is to use `prettier-ignore`.

![Screenshot 2020-09-01 at 2 20 13 PM](https://user-images.githubusercontent.com/6321840/91828832-452ed700-ec5e-11ea-9f19-494684a73839.png)

**Comment by kaisermann** [comment]
That was indeed an issue! Self-closing tags weren't correctly handled. Fixed in `v4.2.1` 🎉
